op-chain-ops/genesis/genesis.go

```diff
@@ -73,6 +73,9 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 			EIP1559Elasticity:        eip1559Elasticity,
 			EIP1559DenominatorCanyon: eip1559DenomCanyon,
 		},
+		EntryPointAddress:     common.HexToAddress("0x0000000000000000000000000000000000007560"),
```

Comment:
I think it would be nice to add AA_SENDER_CREATOR(address(ffff7560)) while adding the address information~!